packages/loader/driver-utils/src/bufferToString.ts

```diff
@@ -0,0 +1,22 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { IsoBuffer } from "@fluidframework/common-utils";
+
+/**
+ * Convert binary blob to string format
+ *
+ * @param blob - the binary blob
+ * @returns the blob in string format
+ */
+export const bufferToString = (blob: ArrayBufferLike): string => IsoBuffer.from(blob, "utf8").toString("utf8");
```

Comment:
Are you sure IsoBuffer.from(blob, "utf8") & IsoBuffer.from(blob, "base64") below is correct? It does not make sense to me, as we are converting from ArrayBufferLike to IsoBuffer - there is no encoding or conversion at that step, it's buffer to buffer type.
toString() makes sense, as we are converting a buffer to particular string representation.

I believe you want just IsoBuffer.from(blob)